woocommerce/woocommerce-admin
# Modified layout.js withSelect

Fixes #4479 
This PR adds the code necessary to be sure we're always setting a boolean value in `taskListComplete`.

### Screenshots
No warning message should appear.
<img width="1231" alt="screenshot" src="https://user-images.githubusercontent.com/1314156/83449127-74e4fd80-a429-11ea-952c-73e061d0088c.png">

### Detailed test instructions:
- Go to the Homepage (URL `/wp-admin/admin.php?page=wc-admin`)
- Verify there is no warning message in the browser console.

You can check whether it continues working with a `false` and `true` value. To do this:
- Execute the following SQL in your database:
````
UPDATE `wp_options` SET `option_value` = '0' WHERE `option_name` = 'woocommerce_task_list_complete';
````


### Changelog Note:
Fix: Fixed the problem that creates a browser console warning on the Homepage.
